docs/_toc.yml

```diff
@@ -61,3 +61,6 @@ parts:
       - file: frontmatter
       - file: settings
       - file: glossary
+  - caption: Contribute
+    chapters:
+      - file: contribute/docs
```

Comment:
Until we fix/implement #670 I think we should keep things flat. This will currently create a URL that is `guide/docs` which is not ideal.

Reply:
OK, will move it back. I guess we can set up redirects to nested folders whenever that functionality is in? Unless you think there's a strong reason to use `category-pagename.md` syntax instead of `category/pagename.md` structure?

Either way let's discuss that one over in [#966](https://github.com/executablebooks/mystmd/issues/966)

Reply:
I think we should absolutely support folders and map that to URLs - which is the feature tracked in #670. Moving the mystmd docs over to use that vs being flat I don't really have a preference on -- as it grows more organization is nice.

For this folder specifically, do you see the `contributing` folder to have more pages?

Reply:
For this one, I assume that we want documentation for newcomers to understand how to develop `mystmd`. That might cover things like "what is remix" "how to learn react" "how to make style changes to a theme" etc. We need comprehensive contributing documentation if we want to grow an open source community. For that kind of content, is there any _other_ location you'd imagine it living?

Reply:
Nice -- the examples help, thanks. Especially showing the `skills` part of the jupyterhub docs.

Reply:
This also makes a lot more sense given that the title of this is `Contribute to the docs` I missed this in my initial review...

![image](https://github.com/executablebooks/mystmd/assets/913249/651ed593-6804-4eef-8c84-7abeb5a8bb2e)